### DISCUSSION TITLE
CI: extend mypy and formating checks to include discopop_wizard

### DISCUSSION THREAD

**Discussion by lukasrothenberger** [discopop-project/discopop#330] [Development]
Currently, only the `explorer` and `profiler` are covered by CI checks for type correctness and code formatting.
- [x] Extend the CI Workflow to cover the `wizard` as well.
- [x] Apply automatic formatting to `wizard` (`python -m black -l 100 discopop_wizard`)
- [x] Fix potential typing errors in `wizard` detected by `mypy` (`python -m mypy -p discopop_wizard`)

**Comment by goerlibe** [comment]
all done already, except for ci formatting of wizard, which is done in #405
